WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -318,7 +318,17 @@ class WebKitViewController: UIViewController, WebKitAuthenticatable {
         guard let toolBar = navigationController?.toolbar else {
             return
         }
-        toolBar.barTintColor = UIColor(light: .white, dark: .appBarBackground)
+
+        let appearance = UIToolbarAppearance()
```

Comment:
Instead of adding an exception for iOS 15 I went ahead and applied these stylings to all iOS versions for consistency. This does mean that iOS 14's "tinting" of the toolbar with a blur effect is gone - it's entirely opaque now.